Lesson14/Task1.py

```diff
@@ -0,0 +1,19 @@
+def logger(func):
+    def wrapper(*args, **kwargs):
+        print(f'{wrapper} called with {args}, {kwargs}')
+        return func(*args, **kwargs)
+    return wrapper
```

Comment:
According to the task, the logger should print the function - `For example: "add called with 4, 5"`

Now it prints wrapper, not function:
`<function logger.<locals>.wrapper at 0x7f9e4f885430> called with (4, 5), {}`